torchtune/models/llama3_2_vision/_model_builders.py

```diff
@@ -75,47 +123,164 @@ def llama3_2_vision_11b(
+        fusion_lora=fusion_type == LoRATrainable.LORA,
+        lora_attn_modules=lora_attn_modules,
+        apply_lora_to_mlp=apply_lora_to_mlp,
+        apply_lora_to_output=apply_lora_to_output,
+        vocab_size=128_256,
+        num_layers=32,
+        fusion_interval=4,
+        num_special_tokens=8,
+        num_heads=32,
+        num_kv_heads=8,
+        embed_dim=4096,
+        max_seq_len=131_072,
+        encoder_max_seq_len=128_080,  # 20*6404
+        rope_base=500000.0,
+        intermediate_dim=14336,
+        lora_rank=lora_rank,
+        lora_alpha=lora_alpha,
+        lora_dropout=lora_dropout,
+        use_dora=use_dora,
+        quantize_base=quantize_base,
+    )
+    return DeepFusionModel(
+        encoder=encoder,
+        decoder=decoder,
+        encoder_trainable=encoder_type != LoRATrainable.FROZEN,
+        decoder_trainable=decoder_type != LoRATrainable.FROZEN,
+        fusion_trainable=fusion_type != LoRATrainable.FROZEN,
+    )
+
+
+def llama3_2_vision_90b(
+    decoder_trainable: bool = False,
+    encoder_trainable: bool = True,
+    fusion_trainable: bool = True,
+    image_size: int = 560,
+) -> DeepFusionModel:
+    """Llama 3.2 Vision 90B model
+
+    Args:
+        decoder_trainable (bool): Whether to make decoder params trainable. Default is False.
+        encoder_trainable (bool): Whether to make encoder params trainable. Default is True.
+        fusion_trainable (bool): Whether to make fusion params trainable. Default is True.
+        image_size (int): Base image size that images will be tiled and resized to.
+            Default is 560 for Instruct weights, use 448 for pre-trained.
+
+    Returns:
+        DeepFusionModel: Instantiation of the Llama 3.2 Vision 90B model
+    """
+    encoder = llama3_2_vision_encoder(
         patch_size=14,
+        num_heads=16,
+        clip_embed_dim=1280,
+        clip_num_layers=32,
+        clip_hidden_states=[3, 7, 15, 23, 30],
+        decoder_embed_dim=8192,
+        num_layers_projection=8,
+        tile_size=image_size,
         max_num_tiles=4,
-        max_seq_len=max_seq_len,
-        image_mean=(0.48145466, 0.4578275, 0.40821073),
-        image_std=(0.26862954, 0.26130258, 0.27577711),
-        prompt_template=template,
+        in_channels=3,
+    )
+    decoder = llama3_2_vision_decoder(
+        vocab_size=128_256,
+        num_layers=100,
+        fusion_interval=4,
+        num_special_tokens=8,
+        num_heads=64,
+        num_kv_heads=8,
+        embed_dim=8192,
+        max_seq_len=131_072,
+        encoder_max_seq_len=128_080,  # 20*6404
+        rope_base=500000.0,
+        intermediate_dim=28672,
+    )
+    return DeepFusionModel(
+        encoder=encoder,
+        decoder=decoder,
+        encoder_trainable=encoder_trainable,
+        decoder_trainable=decoder_trainable,
+        fusion_trainable=fusion_trainable,
     )
 
 
-def lora_llama3_2_vision_11b(
+def lora_llama3_2_vision_90b(
```

Comment:
git is confused :(

lora_llama3_2_vision_11b is still at the top and was not replaced.

this function is a copy of lora_llama3_2_vision_11b